### ISSUE TITLE
Create fun banner

### ISSUE THREAD

**Issue by nicholasericksen** [nicholasericksen/ddos#1]
All exploit tools seem to have cool banners, might as well have one here too. Maybe an ascii sheep lol.

**Comment by nicholasericksen** [comment]
maybe just a ton of these little ones 

```
,mmm^>         
 W-W' 
```

**Comment by nicholasericksen** [comment]
reminds me of the good old sheep bomb back in the lync days .. good ol days...bah

**Comment by nicholasericksen** [comment]
It worked out in the end. Implemented here for `v0.2`: https://github.com/nicholasericksen/ddos/blob/81794fbae4281198c223d9afc3fde2f5ecf97ee5/main.go#L36

Preview below:
```
                 ,mmm^>
   \|  ddos  __   W-W'  ___

```
